server/routes/students.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express')
+const { createStudent, getStudent, getStudents, deleteStudent, updateStudent } = require('../controllers/studentController')
+const student = require('../models/studentModel')
```

Comment:
just fyi - you can use import instead of require as well. More about it [here](https://stackoverflow.com/questions/45854169/how-can-i-use-an-es6-import-in-node-js)

Reply:
Yeah thanks. in some courses the old way is still used and import in some other newer courses. I am gonna use them..